spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/dsl/DestinationNode.java

```diff
@@ -56,7 +56,7 @@ public String stringify(boolean includePositionalInfo) {
 
 	@Override
 	public String toString() {
-		return getDestinationName();
+		return ":" + getDestinationName();
```

Comment:
I think we can settle this here.  Seems like a nice little tweak but unrelated.

Removing the `:` has the newly added test `StreamParserTests.testGetStreamData` failing 

```
		node = parse (":fooStream > bar");
		assertThat( node.getStreamData(), equalTo(":fooStream > bar"));
```
```
Expected ::fooStream > bar
Actual   :fooStream > bar
```

The method `getStreamData` was created a while ago and delegates to the `toString` method for all the nodes, looks like the sort of `toDSLString` method any mentioned.  As the `getStreamData` method wasn't called at all before, I think we are safe making this change.